ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -66,48 +67,51 @@ def check_dtl(logger):
             logger.skip("No vdisks present in cluster.", test_name)
 
     @staticmethod
-    @timeout_decorator.timeout(15)
-    def _check_volumedriver(volume_name, vpool_guid, volume_size=VDISK_CHECK_SIZE):
+    @timeout_decorator.timeout(30)
+    def _check_volumedriver(vdisk_name, storagedriver_guid, vpool_name, vdisk_size=VDISK_CHECK_SIZE):
         """
-        Async method to checks if a VOLUMEDRIVER `truncate` works on a vpool
-        Always try to check if the file exists after performing this method
-
-        :param volume_name:
-        :param volume_size:
-        :param vpool_guid:
-        :return:
+        Checks if the volumedriver can create a new vdisk
+
+        :param vdisk_name: name of a vdisk (e.g. test.raw)
+        :type vdisk_name: str
+        :param vdisk_size: size of the volume in bytes (e.g. 10737418240 is 10GB in bytes)
+        :type vdisk_size: int
+        :param storagedriver_guid: guid of a storagedriver
+        :type storagedriver_guid: str
+        :return: True if succeeded, False if failed
+        :rtype: bool
         """
-        vpool = VPoolHelper.get_vpool_by_guid(vpool_guid)
-        storagedriver = None
-        for std in vpool.storagedrivers:
-            if VolumedriverHealthCheck.MACHINE_DETAILS.guid == std.storagerouter_guid:
-                storagedriver = std
-                break
-        if storagedriver is None:
-            raise ValueError('Could not find the right storagedriver for storagerouter {0}'.format(VolumedriverHealthCheck.MACHINE_DETAILS.guid))
-        try:
-            return VDiskController.create_new(volume_name, volume_size, storagedriver.guid)
-        except Exception as ex:
-            raise IOError(ex.message)
+
+        VDiskController.create_new(vdisk_name, vdisk_size, storagedriver_guid)
+        return True
 
```

Comment:
Method description says:
'return: True if succeeded, False if failed'. However only true will be returned. No errors are caught. Either remove the return type and expect it can fail or catch it there (and lose the error reference when only returning false)